docs/src/pages/nuxt.mdx

```diff
@@ -0,0 +1,257 @@
+  from `/api/uploadthing` on your application.
+</Callout>
+
+```ts copy filename="server/api/uploadthing.ts"
+import { createNuxtRouteHandler } from "uploadthing/nuxt";
+
+import { uploadRouter } from "../uploadthing";
+
+export default createNuxtRouteHandler({
+  router: uploadRouter,
+});
+```
+
+### Import CSS
+
+You need to import our styles for the button and dropzone to look right
+
+```ts copy filename="nuxt.config.ts"
+export default defineNuxtConfig({
+  css: ["@uploadthing/vue/style.css"],
+});
+```
+
+### Use the FileRouter in your app
+
+The `@uploadthing/nuxt` package includes "useUploader" builder that you can use
+to generate type-safe component, drop it into app and start uploading files.
+
+```vue copy filename="components/Home.vue"
+<script setup lang="ts">
+import type { OurFileRouter } from "server/uploadthing";
+
+import { useUploader } from "@uploadthing/vue";
+
+const Uploader = useUploader<OurFileRouter>();
+</script>
+
+<template>
+  <main
+    class="flex min-h-screen flex-col items-center justify-center gap-16 p-24"
+  >
+    <Uploader :config="{ endpoint: 'videoAndImage', onClientUploadComplete={()
+    => { alert("Upload Completed"); }} onUploadError={(error: Error) => { // Do
+    something with the error. alert(`ERROR! ${error.message}`); }} }" />
+  </main>
+</template>
+```
+
+</Steps>
+
+### Use the FileRouter in your app
+
+#### Uploader
+
+The `@uploadthing/nuxt` package includes "useUploader" builder that you can use
+to generate type-safe component, drop it into app and start uploading files.
+
+```vue copy filename="components/Home.vue"
+<script setup lang="ts">
+import type { OurFileRouter } from "server/uploadthing";
+
+import { useUploader } from "@uploadthing/vue";
+
+const Uploader = useUploader<OurFileRouter>();
+</script>
+
+<template>
+  <main
+    class="flex min-h-screen flex-col items-center justify-center gap-16 p-24"
+  >
+    <ClientOnly fallback-tag="div" fallback="Loading...">
+      <Uploader :config="{ endpoint: 'videoAndImage', onClientUploadComplete={()
+      => { alert("Upload Completed"); }} onUploadError={(error: Error) => { //
+      Do something with the error. alert(`ERROR! ${error.message}`); }} }" />
+    </ClientOnly>
+  </main>
+</template>
+```
+
+#### UploadButton
+
+This is a VueJS button component that will be used to upload files when clicking
+on it.
+
+```vue copy filename="components/Home.vue"
+<script setup lang="ts">
+import type { OurFileRouter } from "server/uploadthing";
+
+import { useUploadButton } from "@uploadthing/vue";
+
+const UploadButton = useUploadButton<OurFileRouter>();
+</script>
+
+<template>
+  <main
+    class="flex min-h-screen flex-col items-center justify-center gap-16 p-24"
+  >
+    <UploadButton :config="{ endpoint: 'videoAndImage',
+    onClientUploadComplete={() => { alert("Upload Completed"); }}
+    onUploadError={(error: Error) => { // Do something with the error.
+    alert(`ERROR! ${error.message}`); }} }" />
```

Comment:
here it isn't wrapped in ClientOnly 🤔

Reply:
Oh, outdated example. I made it work with SSR, so now no need in using ClientOnly